distribution/build.gradle

```diff
@@ -43,6 +46,21 @@ buildscript {
 Collection distributions = project.subprojects.findAll {
    it.path.contains(':tools') == false && it.path.contains(':bwc') == false }
 
+/*****************************************************************************
+ *                  Third party dependencies report                          *
+ *****************************************************************************/
+
+// Concatenates the dependencies CSV files into a single file
+task generateDependenciesReport(type: ConcatFilesTask) {
+  files = fileTree(dir: project.rootDir,
+          excludes: ['qa/**/dependencies.csv', 'benchmark/**/dependencies.csv', 'test/**/dependencies.csv'],
```

Comment:
Instead of needing to do excludes, how about disabling the dependencies info task in those projects?

Reply:
Yes, I have added `dependenciesInfo.enabled = false` into projects where licenses are not managed ( `build.gradle` file contains `dependencyLicenses.enabled = false` )